packages/config-manager/README.md

```diff
@@ -3,7 +3,7 @@
 ## Example
 
 ```ts
-import { initializeConfig, predefined } from '@ckb-lumos/config';
+import { initializeConfig, predefined } from '@ckb-lumos/lumos/config';
```

Comment:
```suggestion
import { initializeConfig, predefined } from '@ckb-lumos/config';
```

The README for the child package `@ckb-lumos/config-manager` doesn't need to be changed for the entry package.